src/app/external-cluster-wizard/steps/external-cluster/provider/eks/template.html

```diff
@@ -118,7 +117,7 @@
              [formControlName]="Controls.Vpc" />
     </mat-form-field>
 
-    <ng-container *ngIf="isDialogView() ? true : form.get(Controls.Vpc).value?.main">
+    <ng-container *ngIf="isDialogView() ? true : form.get(Controls.Vpc).value">
```

Comment:
```suggestion
    <ng-container *ngIf="isDialogView() || form.get(Controls.Vpc).value">
```

Reply:
actually i will just make it  `<ng-container *ngIf="form.get(Controls.Vpc).value">` because we already have it in both views and it's only been displayed when VPC have value 
@KhizerRehan right ?

Reply:
right! but i have update in my PR anyways i will resolve issue.

This things is changed in https://github.com/kubermatic/dashboard/pull/4838 PR